### ISSUE TITLE
[BUG] Classes from Tailwind plugin `tailwindcss-padding-safe` not recognised

### ISSUE THREAD

**Issue by dhruvkb** [francoismassart/eslint-plugin-tailwindcss#146]
**Describe the bug**
The classes from `tailwindcss-padding-safe` raise `tailwindcss/no-custom-classname` warning. 

**To Reproduce**
1. Install the `tailwindcss-padding-safe` (from GitHub, not the latest version on `npm`)
    ```json
    "devDependencies": {
      ...
      "tailwindcss-padding-safe": "github:desaintflorent/tailwindcss-padding-safe"
    }
    ```
2. Use the safe padding class somewhere e.g. `py-8-safe` or `lg:py-12-safe`.
4. Run ESLint.
5. See error

<img width="853" alt="Screenshot 2022-06-13 at 8 13 16 AM" src="https://user-images.githubusercontent.com/16580576/173278313-1c663a0d-2c31-4682-a454-a77b17e1e7ee.png">

**Expected behavior**
Since these are utilities added by a Tailwind plugin, I expected not to see this warning.

**Environment (please complete the following information):**
 - OS: macOS
- Package versions:
  - tailwindcss@3.1.2
  - postcss@8.4.14
  - eslint@8.16.0
  - eslint-plugin-tailwindcss@3.5.2

**Comment by francoismassart** [comment]
@dhruvkb please try
`npm i eslint-plugin-tailwindcss@3.6.0-beta.1`

and confirm it is fixing your issue.

Thanks

**Comment by dhruvkb** [comment]
I actually dropped the `tailwindcss-padding-safe` plugin in favour of a couple of custom utilities. We can close this issue.
